tests/integration.rs

```diff
@@ -85,6 +85,25 @@ fn test_syntax_assert_pass_update() {
     );
 }
 
+fn loop_unroll(source: &str) -> Option<u64> {
+    let v: Vec<_> = source.split('\n').collect();
```

Comment:
Can't you directly loop over the iterator without collecting it into a vector?

Reply:
yea, I'll rewrite this function, Thibaut gave me some tips